lib/shared/chips/community_chip.dart

```diff
@@ -15,10 +15,11 @@ import 'package:thunder/utils/instance.dart';
 class CommunityChip extends StatelessWidget {
   const CommunityChip({
     super.key,
-    this.communityId,
-    this.communityAvatar,
-    this.communityName,
-    this.communityUrl,
+    required this.communityId,
```

Comment:
If we're marking these as required, then we should make the parameter's non-nullable below!

Reply:
Sure thing, just changed! (I wouldn't say `required` fields should _always_ be non-nullable; sometimes it's valid to pass `null`, but in this case everything works, so we're good.)